src/include/wiredtiger.in

```diff
@@ -259,6 +259,25 @@ struct __wt_cursor {
 	 */
 	int __F(get_value)(WT_CURSOR *cursor, ...);
 
+        /*!
+	 * Get the raw key and value for the current record.
```

Comment:
This is a bit of a different take on the raw cursors - which are configured via an initial configuration option when creating a cursor. Does it make sense to update the `Raw mode` documentation in src/docs/cursors.dox` to cover this new API as well?

Reply:
Done!